### PR TITLE
Fix #2430-app-screen

### LINKED ISSUE #2430: 🐛 Design: App Screen - Empty State
### Feature description

We need to replace our current "no app" screen attached to the one we currently have

image

This is what we want it to be:

image

The image:
https://www.figma.com/file/tIfcueoxU4DtYtCHmmLuE7/Community---Task-Assets?node-id=1%3A2096

### Use case

_No response_

### Are you willing to submit PR?

_No response_

### PR DESCRIPTION
**Pull request by rkshaw20**
Issue Number: #2430

## PR Checklist
- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/code_of_conduct.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-client/src/Components/SvgThemeImage.scss [hunk 1]
```diff
@@ -1,8 +1,8 @@
 @import "../style/index.scss";
 
 .svg-theme-image {
-  width: 150px;
-  height: 150px;
+  width: 250px;
+  height: 250px;
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Components/SvgThemeImage.scss:5]: Changing the default size of "SvgThemeImage" will impact other places and not only the app list page

Please do one of the following:

1. Add property to the SvgThemeImage component for size (something like small/medium/large) and set the size in CSS based on the size property using a new class

2. override the size of the image in the ApplicationList.scss 

.application-list {
  &__empty-state {
     .svg-theme-image {
        width: 250px;
         height: 250px;
     }
   }
}

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]
@rkshaw20  congrats on your first PR with Amplication
please see my comments

File: packages/amplication-client/src/Components/SvgThemeImage.scss (hunk 1 above)
Comment (rkshaw20) [reply packages/amplication-client/src/Components/SvgThemeImage.scss:5]: okay, doing the changes.

**Review by yuval-hazaz — APPROVED** [review]

**Comment by yuval-hazaz** [comment]
@mshidlov please merge into the next release

**Comment by tupe12334** [comment]
The font size of the title is 24px and in the figma is 18px @rkshaw20

**Comment by rkshaw20** [comment]
> The font size of the title is 24px and in the figma is 18px and the text container is div and not span and the text is not in the image box, you can give it -50 margin bottom @rkshaw20

okay, doing

**Comment by rkshaw20** [comment]
Hey @tupe12334 , I have updated the changes. Please check it.

**Comment by rkshaw20** [comment]
> Hey @tupe12334 , I have updated the changes. Please check it.

Hey @tupe12334 , you have checked my changes? Please check it , if not checked yet.

**Comment by mshidlov** [comment]
@rkshaw20 thank you will review it soon, this PR is planed to be merged to the next release

**Comment by tupe12334** [comment]
@rkshaw20 I saw your changes but we need to make it look exactly like the styling by @alexbass86

**Comment by yuval-hazaz** [comment]
@rkshaw20 Thanks for the great work!
@tupe12334 I will merge this PR into a new branch. If there are any additional changes that are requested by @alexbass86 please complete them soon, so we can merge the new branch to next release

**Comment by rkshaw20** [comment]
> @rkshaw20 Thanks for the great work! @tupe12334 I will merge this PR into a new branch. If there are any additional changes that are requested by @alexbass86 please complete them soon, so we can merge the new branch to next release

Sure, I will do it ASAP @yuval-hazaz @tupe12334

**Comment by rkshaw20** [comment]
I have made the changes. Where I should add them?

**Pull request merged**